app/helpers/sort_helper.rb

```diff
@@ -270,4 +278,14 @@ def sort_header_tag(column, options = {})
 
     content_tag('th', sort_link(column, caption, default_order, lang: lang), options)
   end
+
+  # Returns a table header tag similar to +sort_header_tag+, but
+  # according to the LSG table component.
+  def sort_header_tag_with_lsg(column, options = {})
```

Comment:
Yeah, that's just a lovely temporary hack(tm) . @ulferts and @HDinger are already working on updating the default sort header to use the LSG style over at https://github.com/opf/openproject/pull/3343